### ISSUE TITLE
[pkg] v0.3.12 pypi is not updated

### ISSUE THREAD

**Issue by xmnlab** [stanfordmlgroup/ngboost#283]
Hi everyone!

I am trying to use v0.3.12 but it seems that the package on pypi is not the same as the code here on github under tag v0.3.12

I downloaded the package from pypi and the init function is:

```python
class NGBRegressor(NGBoost, BaseEstimator):
    """
    Constructor for NGBoost regression models.

    NGBRegressor is a wrapper for the generic NGBoost class that facilitates regression.
    Use this class if you want to predict an outcome that could take an
    infinite number of (ordered) values.

    Parameters:
        Dist              : assumed distributional form of Y|X=x.
                            A distribution from ngboost.distns, e.g. Normal
        Score             : rule to compare probabilistic predictions P̂ to the observed data y.
                            A score from ngboost.scores, e.g. LogScore
        Base              : base learner to use in the boosting algorithm.
                            Any instantiated sklearn regressor, e.g. DecisionTreeRegressor()
        natural_gradient  : logical flag indicating whether the natural gradient should be used
        n_estimators      : the number of boosting iterations to fit
        learning_rate     : the learning rate
        minibatch_frac    : the percent subsample of rows to use in each boosting iteration
        col_sample        : the percent subsample of columns to use in each boosting iteration
        verbose           : flag indicating whether output should be printed during fitting
        verbose_eval      : increment (in boosting iterations) at which output should be printed
        tol               : numerical tolerance to be used in optimization
        random_state      : seed for reproducibility. See
                            https://stackoverflow.com/questions/28064634/random-state-pseudo-random-number-in-scikit-learn
    Output:
        An NGBRegressor object that can be fit.
    """

    def __init__(
        self,
        Dist=Normal,
        Score=LogScore,
        Base=default_tree_learner,
        natural_gradient=True,
        n_estimators=500,
        learning_rate=0.01,
        minibatch_frac=1.0,
        col_sample=1.0,
        verbose=True,
        verbose_eval=100,
        tol=1e-4,
        random_state=None,
    ):
```

but the same code under tag v0.3.12 is different: https://github.com/stanfordmlgroup/ngboost/blob/v0.3.12/ngboost/api.py#L21-L73

not sure if I am missing something .. but it looks like it is not updated.

any change to update the package on pypi?

all the best! thanks!

**Comment by xmnlab** [comment]
Hi everyone, just a friendly reminder about this issue :)

**Comment by xmnlab** [comment]
Hi folks, just a friendly reminder about this issue if there is any update

**Comment by xmnlab** [comment]
hi everyone! just checking if there is any update about this issue. thanks!

**Comment by xmnlab** [comment]
hi everyone! just a friendly reminder about this issue. let me know if there is anything I could help. thanks!!

**Comment by xmnlab** [comment]
because we didn't have any progress here since February and I've had received any response since July, 
for now I will publish a new package on pypi (not sure yet the name) just in order to have this available on pypi because currently it is causing on trouble. thanks you all for all the work on ngboost. 

If I can help in any way here feel free to ping me anytime :)

**Comment by xmnlab** [comment]
https://pypi.org/project/ngboost-release/

**Comment by xmnlab** [comment]
thank you so much for the update! appreciate that!
looking forward to using the new release next week :)

**Comment by ryan-wolbeck** [comment]
we are working on https://github.com/stanfordmlgroup/ngboost/pull/296 and I'll publish 3.13 once merged

**Comment by ryan-wolbeck** [comment]
we published 0.3.13 and verified this was resolved

**Comment by xmnlab** [comment]
Thank you so much!!
